wgpu/backends/rs.py

```diff
@@ -862,7 +880,8 @@ def create_render_pipeline(
             assert (
                 depth_stencil.get("format", None) is not None
             ), "depth_stencil needs format"
-            stencil_front = depth_stencil.get("front", {})
+            stencil_front = depth_stencil.get("stencil_front", {})
```

Comment:
I changed this from stencil_front -> front earlier, I don't know why I thought this was changed.